pkgs/applications/science/math/geogebra/default.nix

```diff
@@ -31,10 +31,10 @@ let
     license = with licenses; [ gpl3 cc-by-nc-sa-30 geogebra ];
     sourceProvenance = with sourceTypes; [
       binaryBytecode
-      binaryNativeCode  # some jars include native binaries
+      binaryNativeCode # some jars include native binaries
     ];
     platforms = with platforms; linux ++ darwin;
-    hydraPlatforms = [];
+    hydraPlatforms = [ ];
```

Comment:
Please do not change layout in an update commit. As this messes up `git blame` if merged.

Reply:
Already restored the layout in the new commit.